File: 4-semestre/pesquisa e ordenação/aula 02 - desafio arquivo/python/principal.py
from util import Util

lista_numeros = []
lista2_numeros = []

lista_palavras = []

# Util.popular_aleatorio_numeros(lista_numeros, 10, 100, 500)
# Util.exibir_lista(lista_numeros)

#lista2_numeros.extend(lista_numeros)

#Util.popular_aleatorio_palavras(lista_palavras, 10, 5)
# Util.exibir_lista(lista_palavras)

lista_num = Util.popular_numeros_arquivo('numeros.txt')
lista_p = Util.popular_palavras_arquivo('palavras.txt')

print('Lista números:')
Util.exibir_lista(lista_num)
print('\nLista palavras:')
Util.exibir_lista(lista_p)

File: 4-semestre/pesquisa e ordenação/aula 02 - desafio arquivo/java/Principal.java
package pkg;

import java.util.ArrayList;
import java.util.List;

public class Principal {
	public static void main(String[] args) {
		ArrayList<Integer> listaNumeros = new ArrayList<>();
        ArrayList<Integer> lista2Numeros = new ArrayList<>();
        ArrayList<String> listaPalavras = new ArrayList<>();
        
        Util.popularAleatorioNumeros(listaNumeros, 10, 100, 500);
        Util.exibirListaNumeros(listaNumeros);
        
        lista2Numeros.addAll(listaNumeros);
        
        Util.popularAleatorioPalavras(listaPalavras, 10, 5);
        Util.exibirListaPalavras(listaPalavras);

		ArrayList<String> listaPalavrasArquivo = new ArrayList<>();
		ArrayList<String> listaNumerosArquivo = new ArrayList<>();

		listaPalavrasArquivo = Util.("palavras");
		Util.exibirListaPalavras(listaPalavrasArquivo);
		listaNumerosArquivo = Util.("numeros");
		Util.exibirListaPalavras(listaPalavrasArquivo);
	}
}


